tools/keytools/keygen.c

```diff
@@ -1402,6 +1409,9 @@ int main(int argc, char** argv)
         else if (strcmp(argv[i], "--force") == 0) {
             force = 1;
         }
+        else if (strcmp(argv[i], "--no-overwrite") == 0) {
```

Comment:
Please add keygen argument to docs/Signing.md. That document is the only place the arguments are found.